docs/roman/datamodels/metadata.rst

```diff
@@ -84,101 +54,15 @@ databases, such as CRDS, where the path to the metadata element is
-    >>> trans.coeff = 42.0
-    >>> model.meta.transformations.append(trans)
-    >>> model.meta.transformations.append({'type': 'SIN', 'coeff': 42.0})
-
-When accessing the items of the list, the result is a normal metadata
-object where the attributes are type-checked::
-
-.. doctest-skip::
-
-    >>> trans = model.meta.transformations[0]
-    >>> print(trans)
-    <romancal.datamodels.schema.Transformations object at 0x123a810>
-    >>> print(trans.type)
-    SIN
-    >>> trans.type = 42.0
-    Traceback (most recent call last):
-      File "<stdin>", line 1, in <module>
-      File "site-packages/romancal.datamodels/schema.py", line 672, in __setattr__
-         object.__setattr__(self, attr, val)
-      File "site-packages/romancal.datamodels/schema.py", line 490, in __set__
-         val = self.to_basic_type(val)
-      File "site-packages/romancal.datamodels/schema.py", line 422, in to_basic_type
-         raise ValueError(e.message)
-    ValueError: 42.0 is not of type u'string'
-
-JSON Schema
-===========
-
-The `romancal.datamodels` library defines its metadata using `Draft 4 of
-the JSON Schema specification
-<http://tools.ietf.org/html/draft-zyp-json-schema-04>`_, but
-romancal.datamodels uses YAML for the syntax.  A good resource for
-learning about JSON schema is the book `Understanding JSON Schema
-<http://spacetelescope.github.com/understanding-json-schema>`_.  The
-mapping from Javascript to Python concepts (such as Javascript “array”
-== Python “list”) is added where applicable.
-
-In addition to the standard JSON Schema metadata, ``romancal.datamodels``
-also supports the following additional metadata.
-
-Arrays
-''''''
-
-The following metadata have to do with validating n-dimensional arrays:
-
-- ``ndim``: The number of dimensions of the array.
-
-- ``max_ndim``: The maximum number of dimensions of the array.
-
-- ``datatype``: For defining an array, ``datatype`` should be a string.
-  For defining a table, it should be a list.
-
-- **array**: ``datatype`` should be one of the following strings,
-  representing fixed-length datatypes:
-
-  bool8, int8, int16, int32, int64, uint8, uint16, uint32,
-  uint64, float16, float32, float64, float128, complex64,
-  complex128, complex256
-
-Or, for fixed-length strings, an array ``[ascii, XX]`` where
-``XX`` is the maximum length of the string.
-
-(Datatypes whose size depend on the platform are not supported
-since this would make files less portable).
-
-- **table**: ``datatype`` should be a list of dictionaries.  Each
-  element in the list defines a column and has the following keys:
-
-  - ``datatype``: A string to select the type of the column.
-    This is the same as the ``datatype`` for an array (as
-    described above).
+    print(raw_science.meta['observation']['start_time'])
+    print(raw_science.meta.observation.start_time)
 
-  - ``name`` (optional): An optional name for the column.
+In addtion the times are stored as python time objects and so the date can be
```

Comment:
Typo in `addtion`

Reply:
Maybe `astropy Time` objects, since the Python standard library has its own `time` class with different methods?